bin/test-lxd-network

```diff
@@ -257,11 +269,15 @@ lxc start v1-physical
 waitVMAgent v1-physical
 
 # Interface "eth0" should exist in the VM with the correct MTU.
-lxc exec v1-physical -- ip link show eth0
+lxc exec v1-physical -- ls /sys/class/net/eth0
 lxc exec v1-physical -- grep -Fx 1400 /sys/class/net/eth0/mtu
 
+# Check VM bridged multi-queue support.
+lxc exec v1-physical -- apt-get install ethtool --yes
+lxc exec v1-physical -- ethtool -l eth0 | grep -c 'Combined:\s\+3'  | grep -Fx 2
+
 # Default VM interface enp5s0 should not exist in the VM.
-! lxc exec v1-physical -- ip link show enp5s0 || false
+! lxc exec v1-physical -- ls /sys/class/net/enp5s0 || false
```

Comment:
Just a suggestion but, `test -d` might be less noisy.

Reply:
fixed